app/pages/shared_documents/renderDocument.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+function renderDocument(SharedFile $shared_doc)
+{
+    $file_mime = $shared_doc->mime;
+
+    $file_icon = match ($file_mime) {
+        'pdf' => 'fas fa-file-pdf',
+        'png', 'jpg', 'jpeg', 'gif' => 'fas fa-file-image',
+        'doc', 'docx' => 'fas fa-file-word',
+        'xls', 'xlsx' => 'fas fa-file-excel',
+        'ppt', 'pptx' => 'fas fa-file-powerpoint',
+        default => 'fas fa-file',
+    };
```

Comment:
`match` is a very powerful operator. 7 lines of codes compared to 25 with the previous `switch` code